RecoMET/METAlgorithms/src/METSignificance.cc

```diff
@@ -38,7 +38,7 @@ metsig::METSignificance::~METSignificance() {
 reco::METCovMatrix
 metsig::METSignificance::getCovariance(const edm::View<reco::Jet>& jets,
 				       const std::vector< edm::Handle<reco::CandidateView> >& leptons,
-				       const edm::View<reco::Candidate>& pfCandidates,
+				       const edm::Handle<edm::View<reco::Candidate> >& pfCandidates,
```

Comment:
Get the product reference in the function implementation early and use it instead of explicit dereference of the handle.